metrics/default-mobile-app-metrics.md

```diff
@@ -0,0 +1,142 @@
+# Abstract
+
+This document contains the proposal for gathering some default metrics of mobile apps and making them available to developers.
+
+## Goals
+
+### Goals
+
+- Provide mobile app developers a service that can receive some basic default metrics, namely sdk version, app version and platform version
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions
+
+### Non Goals
+
+- Identify individual devices or users in metrics events (bonus)
+
+- Registering of user-defined metrics and logs (not prioritized; just kept in mind)
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service (separate epic/proposal)
+
+## Problem Description
+
+As mobile apps are developed, improved and iterated on, different versions of those apps can be used on many different devices. 
+Metrics data becomes invaluable for developers to observe the landscape of application installations and make sure further 
+development and deployment of backend services will not affect users.
+
+## Proposed Solution
+
+A Metrics Service will be available from the Service Catalog, which can be provisioned by developers for use with their mobile applications.
```

Comment:
Are we talking here about extending the existing Metrics or creating a new one for mobile?

Reply:
For MVP, we're talking about bundling them into single APB.

Reply:
I'm obviously too late for this conversation but if we plan to have 2 different services, wouldn't it be simpler/quicker to have them separated from the start?

Reply:
@david-martin any comments on this?

Reply:
@josemigallas I think getting the mvp with a single APB for both service & mobile app metrics is OK. If there's a push/requirement to separate them into individual APBs, we can address that then.